install_files/securedrop-app-code/DEBIAN/postinst

```diff
@@ -24,8 +24,17 @@ case "$1" in
     chown www-data:www-data /var/www/document.wsgi
     chown www-data:www-data /var/www/source.wsgi
 
-    aa-complain /etc/apparmor.d/usr.sbin.tor
-    aa-complain /etc/apparmor.d/usr.sbin.apache2
+    a2dissite 000-default
```

Comment:
we want the apparmor profile to be part of the app code deb package so we can update it when the application code changes without requiring the playbook to be ran manually by the admin.